src/backend/clearboard/blackNwhite.py

```diff
@@ -0,0 +1,25 @@
+import cv2
```

Comment:
All the python code should go to `/src/backend/clearboard` where you can make a folder for all opencv scripts.